### PR TITLE
Fix validating callback type for distributed Keras

### PR DESCRIPTION
**Pull request by EFanZh**

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thank you for your PR. Could you please add a quick unit test?

**Comment by EFanZh** [comment]
@fchollet I have added some unit tests.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by caisq** [comment]
@fchollet Can you review this again? Thanks.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rthadur** [comment]
@EFanZh please resolve conflicts

**Review by rchao — CHANGES_REQUESTED** [review]
Thanks for the PR! To move forward, we'd like to request the following changes:

1) distributed_training_utils.py has been moved to tensorflow/python/keras/distribute, so let's sync and apply the change there
2) Add a test target in BUILD file

File: tensorflow/python/keras/distribute/distributed_training_utils_test.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for distributed training utility functions."""
+
+from __future__ import absolute_import, division, print_function
```
Comment (rchao) [inline tensorflow/python/keras/distribute/distributed_training_utils_test.py]: Please avoid multiple import and have them on separate lines. Thank you!

**Review by rchao — CHANGES_REQUESTED** [review]

**Review by rchao — APPROVED** [review]

**Comment by rchao** [comment]
Hello, the content of this PR has been added in https://github.com/tensorflow/tensorflow/commit/c89c25b1f82394ebcefb7bf14dd8e96791aab328#diff-f107d83c80db7670b1808be2ebbf9999 and thus I'll close this PR. Thank you for the contribution.
